specs/rust/cargo_add_git.yaml

```diff
@@ -0,0 +1,14 @@
+---
+name: Add a git dependency to a Rust project
+command: cargo add --git {{url}}
+tags:
+  - rust
+  - cargo
+  - add
+  - dependency
+  - git
```

Comment:
Just to make these tags a little more focused (since they appear within the Warp UI), could we limit these tags to "rust" and "cargo"?